openzwave-sys/openzwave-c/value_classes/value_id.h

```diff
@@ -8,29 +8,38 @@ typedef OpenZWave::ValueID ValueID;
 typedef ValueID::ValueGenre ValueGenre;
 typedef ValueID::ValueType ValueType;
 
+typedef struct cValueID {
```

Comment:
Right now we have 3 separate structures:

1 - ValueID from src/value_classes/ValueID.h. This is pure rust and can have impl methods applied to it
```
#[derive(Clone, Hash)]
pub struct ValueID {
    pub home_id: u32,
    pub id: u64
}
```

2 - cValueID from openzwave-sys/openzwave-c/value_classes/value_id.h 
```
typedef struct cValueID {
    uint32  home_id;
    uint64  id;

    cValueID(uint32 home_id, uint64 id)
      : home_id(home_id), id(id) {}
} cValueID;
``
with the rust side equivalent
```
#[repr(C)]
pub struct ValueID {
  pub home_id: uint32_t,
  pub id: uint64_t
}
```
from openzwave-sys/srv/value_id_classes/valid_id.rs

3 - ValueID class from the OpenZWave C++ code.

cValueID is used by the C code to convert the rust API into the C++ API. I think we can eliminate it entirely.